Build fails on examples/parser.rs with rust-1.66.0

```
error[E0432]: unresolved import `anes::parser`
 --> examples/parser.rs:6:5
  |
6 |     parser::{KeyCode, Parser, Sequence},
  |     ^^^^^^ could not find `parser` in `anes`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `anes` due to previous error
```

--- a/anes/Cargo.toml
+++ b/anes/Cargo.toml
@@ -35,3 +35,7 @@ libc = "0.2.66"
 name = "bench_main"
 harness = false
 required-features = ["parser"]
+
+[[example]]
+name = "parser"
+required-features = ["parser"]

